closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -1037,20 +1076,36 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_DigestComputeAnd
  * Method:    DigestReset
  * Signature: (J)V
```

Comment:
Please update the signature to reflect the fact that this function now returns a value.

Reply:
Updated `DigestReset` signature to be `Signature: (J)I` to reflect that we are now returning an integer.